pom.xml

```diff
@@ -70,8 +70,8 @@
       <module>server/integration</module>
       <module>as-modules/embedded</module>
       <module>as-modules/client</module>
+      <module>integrationtests</module>
       <module>integrationtests/as-integration-embedded</module>
-      <module>integrationtests/as-integration-client</module>
```

Comment:
Actually I think you are right - this should stay here. Let me double check this (in the meantime setting label changes required).